hack/aks/Makefile

```diff
@@ -86,6 +88,65 @@ byocni-up: swift-byocni-up ## Alias to swift-byocni-up
 cilium-up: swift-cilium-up ## Alias to swift-cilium-up
 up: swift-up ## Alias to swift-up
 
+overlay-byocni-mariner-up: rg-up overlay-net-up ## Brings up a Mariner Overlay BYO CNI
+	$(AZCLI) aks create -n $(CLUSTER) -g $(GROUP) -l $(REGION) \
+		--node-count $(NODE_COUNT) \
+		--node-vm-size $(VM_SIZE) \
+		--load-balancer-sku basic \
+		--network-plugin none \
+		--network-plugin-mode overlay \
+		--pod-cidr 192.168.0.0/16 \
+		--vnet-subnet-id /subscriptions/$(SUB)/resourceGroups/$(GROUP)/providers/Microsoft.Network/virtualNetworks/$(VNET)/subnets/nodenet \
+		--no-ssh-key \
+		--os-sku $(MARINER_OS_SKU) \
+		--yes
+	@$(MAKE) set-kubeconf
+
+overlay-byocni-ubuntu22-up: rg-up overlay-net-up ## Brings up an Ubuntu22 Overlay BYO CNI
+	$(AZCLI) aks create -n $(CLUSTER) -g $(GROUP) -l $(REGION) \
+		--node-count $(NODE_COUNT) \
+		--node-vm-size $(VM_SIZE) \
+		--load-balancer-sku basic \
+		--network-plugin none \
+		--network-plugin-mode overlay \
+		--pod-cidr 192.168.0.0/16 \
+		--vnet-subnet-id /subscriptions/$(SUB)/resourceGroups/$(GROUP)/providers/Microsoft.Network/virtualNetworks/$(VNET)/subnets/nodenet \
+		--no-ssh-key \
+		--kubernetes-version $(K8S_VER) \
+		--os-sku $(OS_SKU) \
```

Comment:
there's no need for these to be separate new targets, is there? we should add `--os-sku=$(OS_SKU)` to the existing targets, and set the env var when we invoke Make

Reply:
I agree, I made some quick changes to learn how the pipeline works so this comment was premature relative to my intended changes. I believe that the current code aligns with your comment.